README.md

```diff
@@ -36,40 +36,52 @@ pip is already installed if you are using Python 2 >=2.7.9 or Python 3 >=3.4 dow
 
 #### Installing with get-pip.py
 
-To install pip with `curl`, execute the following command. Alternatively you can download `get-pip.py` by clicking [here](https://bootstrap.pypa.io/get-pip.py). 
+To install pip with `curl`, execute the following command. Alternatively you can download `get-pip.py` by clicking [here](https://bootstrap.pypa.io/get-pip.py).
 
 ```bash
-$ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
 ```
 
 Then run the following command in the folder where you have downloaded get-pip.py
 
 ```bash
-$ python get-pip.py
+python get-pip.py
 ```
 
 ## Run project locally (Dev Mode)
 
 **Clone the repo**
 
 ```bash
-$ git clone https://github.com/wso2/docs-is.git
+git clone https://github.com/wso2/docs-is.git
 ```
 
 **Install the dependencies**
 
 ```bash
-$ cd docs-is && pip install -r requirements.txt
+cd docs-is && pip install -r requirements.txt
 ```
 
 **Run mkdocs**
 
 Execute the following command from inside the `<Lang folder>`.
 
 ```bash
-$ cd en && mkdocs serve
+cd en && mkdocs serve
 ```
 
+## Troubleshooting
```

Comment:
does this work with the IS 6.0.0 branch as well?

Reply:
Nope. This is only relevant for the 5.11.o branch.